tests/src/usb_testing-udc_stub.ads

```diff
@@ -87,7 +87,19 @@ package USB_Testing.UDC_Stub is
    function Request_Buffer (This          : in out Controller;
```

Comment:
I wonder if it would be better to allocate the buffer in this function instead.
Like it's done in TinyUSB for instance: https://github.com/hathach/tinyusb/blob/fd2cd3fc230fd81f2fba7b9c8eacb0e14b4e042d/src/portable/raspberrypi/rp2040/dcd_rp2040.c#L93